supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -383,29 +380,29 @@ class JesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
     }
   }
 
+  // If one exists, extract the JES error code (not the google RPC) from the error message
+  private[jes] def getJesErrorCode(errorMessage: String): Option[Int] = {
+    Try { errorMessage.substring(0, errorMessage.indexOf(':')).toInt } toOption
+  }
+
   override def handleExecutionFailure(runStatus: RunStatus,
                                       handle: StandardAsyncPendingExecutionHandle,
                                       returnCode: Option[Int]): Future[ExecutionHandle] = {
-    // If one exists, extract the JES error code (not the google RPC) from the error message
-    def getJesErrorCode(errorMessage: String): Option[Int] = {
-      Try { errorMessage.substring(0, errorMessage.indexOf(':')).toInt } toOption
-    }
-
-    val failedStatus: UnsuccessfulRunStatus = runStatus match {
-      case failedStatus: RunStatus.Failed => failedStatus
-      case preemptedStatus: RunStatus.Preempted => preemptedStatus
+    runStatus match {
+      case preemptedStatus: RunStatus.Preempted if preemptible => handlePreemption(preemptedStatus, returnCode)
+      case failedStatus: RunStatus.UnsuccessfulRunStatus => handleFailedRunStatus(failedStatus, handle, returnCode)
       case unknown => throw new RuntimeException(s"handleExecutionFailure not called with RunStatus.Failed or RunStatus.Preempted. Instead got $unknown")
```

Comment:
On the last part, I think what it was is that gotc had seen instances where something had been preempted but received the more general error code (`13` instead of `14` I think, or the other way around). We decided to simply treat those as errors as after talking to Google there was no perfect way to know "is this *really* preempted" in a programatic fashion.